models/Product.js

```diff
@@ -85,6 +117,48 @@ class Product {
       ],
     });
   }
+
+  validateAttributes(product) {
+    if (!Array.isArray(product)) {
+      return this.validateProduct(product);
+    } else {
+      for (let i = 0; i < product.length; i++) {
```

Comment:
Para esta función, se puede usar un forEach o incluso un metodo que se llama .some() en lugar del for tradicional.